examples/kubernetes/README.md

```diff
@@ -1,7 +1,11 @@
 This example runs up a local KIND cluster (3 worker nodes), deploys Couchbase to it using the CAO and then runs up our monitoring stack.
 
-An ingress is created to allow you to access it all via http://localhost/
+By default, it gives you the command to handle port-forwarding to the CMOS service on http://localhost:8080/.
+
+An ingress is optionally created (`SKIP_INGRESS=no`) to allow you to access it all via http://localhost/ but this can trigger forwarding issues in some of the CMOS documentation.
```

Comment:
Does it default to on or off? "optionally created" suggests off, but `SKIP_INGRESS=no` suggests on.